packages/core/src/contrib/CurveConstraints.ts

```diff
@@ -74,7 +67,8 @@ export const curvature = (
   //     DOI: 10.1090/noti1578
   const v1 = ops.vsub(p2, p1);
   const v2 = ops.vsub(p3, p2);
-  const angle = ops.angleFrom(v1, v2);
+  const angle =
+    p1.length === 2 ? ops.angleFrom(v1, v2) : ops.angleBetween(v1, v2);
```

Comment:
Why don't you just always use `angleBetween` here?
```suggestion
  const angle = ops.angleBetween(v1, v2);
```

Reply:
The signed curvature is useful for computing the turning number of planar curves. I added comments to clarify the behaviour of this functions.